pkg/asset/manifests/gcp/loadbalanceruid.go

```diff
@@ -0,0 +1,39 @@
+package gcp
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	gcptypes "github.com/openshift/installer/pkg/types/gcp"
+)
+
+const (
+	// UIDConfigMapName is the Key used to persist UIDs to configmaps.
+	UIDConfigMapName = "ingress-uid"
+
+	// UIDNamespace is the namespace which contains the above config map
+	UIDNamespace = metav1.NamespaceSystem
+
+	// UIDCluster is the data keys for looking up the clusters UID
+	UIDCluster = "uid"
+
+	// UIDProvider is the data keys for looking up the providers UID
+	UIDProvider = "provider-uid"
+)
+
+// GenerateLoadBalancerUIDConfig generates a configmap with a unique UID
+// per cluster used in naming service load balancers created by the
+// cluster's cloud provider.
+func GenerateLoadBalancerUIDConfig(infraID string) *corev1.ConfigMap {
```

Comment:
why the Generate prefix `LoadBalancerUIDConfigMap` seems descriptive enough...

Reply:
https://github.com/openshift/installer/pull/2045#discussion_r305522993 similar here... would be better.